cite.html

```diff
@@ -0,0 +1,178 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="cache-control" content="no-cache" />
+    <title>Citing ASPECT</title>
+    <style type="text/css">
+        .box {
+            background: lightgray;
+	font-family: monospace;
+            width: 75%;
+            padding: 10px;word-wrap: break-word;
+        }
+
+        .editbox {
+            background-color: #5f79b8;
+            width: 75%;
+            padding: 10px;
+        }
+        
+        .note {
+            background: #c18166;
+            width: 75%;
+            font-size: 150%;
+	    margin: auto;
+	    border: 1px solid blue;
+            padding: 5px;
+	text-align: center;
+        }
+    </style>
+</head>
+
+<body>
+
+    <p class="note">NOTE: a link with your selection is available at:<br/>
+        <a style="font-size:75%" id="onlineurl" href="https://geodynamics.github.io/aspect/cite.html">https://geodynamics.github.io/aspect/cite.html</a>
+    </p>
+
+<script src="database.js"></script>
```

Comment:
Can you indent this file in emacs (or somehow else) properly? It's unnecessarily hard to read